sdk/communication/AzureCommunicationCalling/Swift_Generated_Interfaces/ACSStreamSize.h

```diff
@@ -0,0 +1,18 @@
+//+-----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Module name : ACSStreamSize.h
+//
+//------------------------------------------------------------------------------
+
+import Foundation
+
+open class StreamSize : NSObject {
+
+    
+    public init(width: Int32, height: Int32)
+
+    open var width: Int32 { get }
+
+    open var height: Int32 { get }
+}
```

Comment:
Ideally this would be exposed to Swift as a struct rather than a class, if possible

Reply:
Yes good point. I will take a note of this.